rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/KafkasMgmtV1ClientImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.redhat.service.bridge.rhoas;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.openshift.cloud.api.kas.models.ServiceAccount;
+import com.openshift.cloud.api.kas.models.ServiceAccountRequest;
+
+import io.quarkus.oidc.client.NamedOidcClient;
+import io.quarkus.oidc.client.OidcClient;
+import io.quarkus.oidc.client.Tokens;
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.Vertx;
+
+@ApplicationScoped
+public class KafkasMgmtV1ClientImpl extends AbstractAppServicesClientImpl implements KafkasMgmtV1Client {
+
+    private static final Logger LOG = LoggerFactory.getLogger(KafkasMgmtV1ClientImpl.class);
+
+    @ConfigProperty(name = "event-bridge.rhoas.mgmt-api.host")
+    String basePath;
+    @ConfigProperty(name = "event-bridge.rhoas.sso.red-hat.refresh-token")
+    String refreshToken;
+
+    @Inject
+    Vertx vertx;
+
+    @Inject
+    @NamedOidcClient("red-hat-sso")
+    OidcClient client;
+
+    Tokens tokens;
+
+    @PostConstruct
+    void onPostConstruct() {
+        init(vertx);
```

Comment:
@kostola Can the `@Inject` be on the `Vertx` field in the super-class? Doc of `javax.inject.Inject` says:

> Fields and methods in superclasses are injected before those in subclasses

Reply:
Good catch! Fixed :+1: